artemis-protocols/artemis-mqtt-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/mqtt/MQTTProtocolManagerFactory.java

```diff
@@ -41,11 +46,16 @@
    private final Map<String, MQTTConnection> connectedClients  = new ConcurrentHashMap<>();
    private final Map<String, MQTTSessionState> sessionStates = new ConcurrentHashMap<>();
 
+   private long defaultMqttSessionExiryInterval = TransportConstants.DEFAULT_DEFAULT_MQTT_SESSION_EXPIRY_INTERVAL;
+
    @Override
    public ProtocolManager createProtocolManager(ActiveMQServer server,
                                                 final Map<String, Object> parameters,
                                                 List<BaseInterceptor> incomingInterceptors,
                                                 List<BaseInterceptor> outgoingInterceptors) throws Exception {
+      if (parameters.containsKey(TransportConstants.DEFAULT_MQTT_SESSION_EXPIRY_INTERVAL)) {
+         defaultMqttSessionExiryInterval = Long.parseLong((String)parameters.get(TransportConstants.DEFAULT_MQTT_SESSION_EXPIRY_INTERVAL));
```

Comment:
What happens when you have multiple Protocol managers? which one will take the expiryInterval?